cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MathematicalNumericFunctionsTest.scala

```diff
@@ -63,18 +64,20 @@ class MathematicalNumericFunctionsTest extends DocumentingTest {
       }
     }
     section("ceil()", "functions-ceil") {
-      p("`ceil()` returns the smallest integer greater than or equal to the argument.")
-      function("ceil(expression)", ("expression", "A numeric expression."))
+      p("`ceil()` returns the smallest floating point number greater than or equal to a given number, and which is equal to an integer.")
```

Comment:
`"and which is equal to an integer."`
is "equal" the correct word here?

Reply:
Fixed as discussed